src/engine/renderer/tr_scene.cpp

```diff
@@ -684,22 +684,24 @@ so the result may be available delayed.
 */
 qhandle_t RE_RegisterVisTest()
 {
-	int hTest;
-	visTest_t *test;
+	visTest_t *test = nullptr;
```

Comment:
This assignment shouldn't be needed to avoid warnings because it is guaranteed to be assigned in the first iteration of the loop.